### ISSUE TITLE
Get Particle class to work with Unicode, LaTeX, and MathML

### ISSUE THREAD

**Issue by namurphy** [PlasmaPy/PlasmaPy#269]
At present, the `Particle` class pretty much only uses ASCII characters for symbols and aliases.  Here is some typical usage:
```Python
>>> proton = Particle('p+')
>>> proton.particle
'p+'
```
It would be great if we could enable something like the following functionality, where `Particle` accepts unicode aliases and provides the symbol in Unicode, LaTeX, and MathML:
```Python
>>> proton = Particle('p⁺')
>>> proton.Unicode
'p⁺'
>>> proton.LaTeX
'p$^+$'
>>> proton.MathML
<equivalent MathML code>
```
We will need to have a screen reader (text-to-speech) accessible format (which is probably MathML).  I'm also wondering how we can put metadata into an expression in order to provide meaning behind the symbols (e.g., if the MathML code can have a caption or something to define what the variable represents).

This issue is a subset of #176.  The API that comes out of #176 should be identical to (or at least very similar to) the API that comes out of this pull request.  

We may want to wait to do anything major until after the version 0.1.0 release since the feature freeze is happening as soon, and that could end up with us doing some unpleasant rebasing.  We could do a few of the simpler things like putting Unicode aliases for special particles in `parsing.py` after #242 is merged, which is likely minor enough to put in during alpha releases.
